src/Cli/dotnet/Commands/Workload/Install/NetSdkMsiInstallerServer.cs

```diff
@@ -191,7 +191,7 @@ public static NetSdkMsiInstallerServer Create(bool verifySignatures)
         string pipeName = WindowsUtils.CreatePipeName(CurrentProcess.Id);
         NamedPipeServerStream serverPipe = NamedPipeServerStreamAcl.Create(pipeName, PipeDirection.InOut, 1, PipeTransmissionMode.Message,
             PipeOptions.None, 65535, 65535, pipeSecurity);
-        InstallMessageDispatcher dispatcher = new(serverPipe);
+        _ = new InstallMessageDispatcher(serverPipe);
```

Comment:
🤔

Reply:
InstallMessageDispatcher's constructor doesn't seem to do anything interesting, so if we aren't storing this, we should be able to completely delete it.

Reply:
Removed.